engines/groovie/cursor.cpp

```diff
@@ -384,7 +384,7 @@ GrvCursorMan_v2::GrvCursorMan_v2(OSystem *system) :
 
 	// Open the icons file
 	Common::File iconsFile;
-	if (!iconsFile.open("icons.ph"))
+	if (!iconsFile.open("icons.ph") && !iconsFile.open("icons.bin"))
 		error("Groovie::Cursor: Couldn't open icons.ph");
```

Comment:
Shouldn't the error message be updated to also include `icons.bin`?

Reply:
hmm yea, maybe it should just say

"Groovie::Cursor: Couldn't open icons.ph or icons.bin"?

Reply:
Either that or just "Couldn't open icons"